### ISSUE TITLE
Save and exit on the review page. If an applicant is not logged in, they will be prompted to create an account

### ISSUE THREAD

**Issue by sijiayam** [civiform/civiform#8779]
# Description
Mocks in https://github.com/civiform/civiform/issues/7935
Related to https://github.com/civiform/civiform/issues/8792

# Done when
- [ ] An applicant has the option to save and exit on the review page
   - [ ] The save and exit button is button styled with outline button
- [ ] An applicant that is not logged in will be prompted with a modal to create an account to save their progress

**Comment by sijiayam** [comment]
Blocked by content for the modal.

**Comment by ssandbekkhaug** [comment]
From Sijia: This is a dev issue that will include a mock for a modal. We are finalizing the content at the moment. If the strings take longer we can file another issue, if it can be resolved this week I'll fold it into this story.

**Comment by sijiayam** [comment]
@velociberry What text would you suggest for this modal that opens up when a guest user clicks on "save and exit" on the review page. We would like to prompt them to create an account to save their progress. I used some language you provided on a different issue for the draft screenshot.

cc @isabelcshaw 

![Image](https://github.com/user-attachments/assets/56e4d5ec-ba44-4690-a162-4bde60cb5098)

**Comment by velociberry** [comment]
Thanks for the mockup!

Suggested revised text:
--------------------------------------------
**To save your progress, create an account**

This will let you save your application now and finish it later
---------------------------------------------

**Comment by haltmar** [comment]
This issue is fixed in Northstar: https://github.com/civiform/civiform/issues/8654

**Comment by sijiayam** [comment]
@shreyachatterjee00 Im going to hold on this issue in case it conflicts with the copy audit suggestions we have.
